nav2_map_server/src/map_io.cpp

```diff
@@ -505,7 +505,7 @@ void tryWriteMapToFile(
             if (map_cell < 0 || 100 < map_cell) {
               q = MaxRGB;
             } else {
-              q = map_cell / 255.0 * MaxRGB;
+              q = map_cell / 100.0 * MaxRGB;
```

Comment:
Similar to the other PR, the ROS 1 version seems to use 255 (see `negate`) which has been in use for 10+ years. This seems OK on face value, but I'm a little weary 

https://github.com/ros-planning/navigation/blob/noetic-devel/map_server/src/image_loader.cpp#L130-L136

Reply:
Though, the `MapMode::Scale` seems to agree with you, so I'm leaning towards merging this. Lets continue to chat in the other PR you submitted and we'll merge these as a pair

Thanks!

Reply:
Hi @SteveMacenski , thanks for your reviews. You are right, the other PR relies on this one. However, the opposite is not true.

In my understanding, this PR fixes a real bug. It could be merged even though discussions are still opened in the other one.

Reply:
I commented on the other thread. Can you update this PR to contain the updates that supersedes https://github.com/ros-navigation/navigation2/pull/4713? Lets just put both of these together, its a simple enough set of code changes to review and are related